css-positioning/css-positioning-quiz-notes.md

```diff
@@ -0,0 +1,48 @@
+# css-positioning-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is the **default** value for the `position` property of HTML elements?
+The element is positioned according to the normal flow of the document. The top, right, bottom, left, and z-index properties have no effect. This is the default value. (static)
+
+- How does setting `position: relative` on an element affect document flow?
+this does not affect the position of surrounding elements.
+
+- How does setting `position: relative` on an element affect where it appears on the page?
+this moves an element from normal flow, shifting it to the top,right bottom,or left of where it would have been placed.
+- How does setting `position: absolute` on an element affect document flow?
+it does not effect  the position of surrounding elements
```

Comment:
It does impact the position of surrounding elements! When you give an element `position: absolute` the sibling elements will adjust accordingly.